lib/l10n/app_ta.arb

```diff
@@ -0,0 +1,428 @@
+	"sleep_timer": "உறக்க நேரம்",
+	"mins": "{minutes} நிமிடங்கள்",
+	"hours": "{hours} மணிநேரங்கள்",
+	"hour": "{hours} மணிநேரம்",
+	"custom_hours": "தனிப்பயன் மணிநேரங்கள்",
+	"logs": "பதிவுகள்",
+	"developers": "உருவாக்குநர்கள்",
+	"not_logged_in": "நீங்கள் உள்நுழையவில்லை",
+	"search_mode": "தேடல் முறை",
+	"audio_source": "ஒலி மூலம்",
+	"ok": "சரி",
+	"failed_to_encrypt": "குறியாக்கம் தோல்வியடைந்தது",
+	"encryption_failed_warning": "Spotube உங்கள் தரவை பாதுகாப்பாக சேமிக்க குறியாக்கத்தைப் பயன்படுத்துகிறது. ஆனால் அவ்வாறு செய்ய முடியவில்லை. எனவே இது பாதுகாப்பற்ற சேமிப்பகத்திற்கு மாறும்\nநீங்கள் லினக்ஸ் பயன்படுத்துகிறீர்கள் என்றால், எந்த ரகசிய சேவையும் (gnome-keyring, kde-wallet, keepassxc போன்றவை) நிறுவப்பட்டுள்ளதா என்பதை உறுதிப்படுத்தவும்",
+	"querying_info": "தகவலைக் கேட்கிறது...",
+	"piped_api_down": "Piped API செயலிழந்துள்ளது",
+	"piped_down_error_instructions": "Piped நிகழ்வு {pipedInstance} தற்போது செயலிழந்துள்ளது\n\nநிகழ்வை மாற்றவும் அல்லது 'API வகை'யை அதிகாரப்பூர்வ YouTube API க்கு மாற்றவும்\n\nமாற்றத்திற்குப் பிறகு பயன்பாட்டை மறுதொடக்கம் செய்வதை உறுதிப்படுத்தவும்",
+	"you_are_offline": "நீங்கள் தற்போது ஆஃப்லைனில் உள்ளீர்கள்",
+	"connection_restored": "உங்கள் இணைய இணைப்பு மீட்டெடுக்கப்பட்டது",
+	"use_system_title_bar": "கணினி தலைப்புப் பட்டியைப் பயன்படுத்தவும்",
+	"crunching_results": "முடிவுகளை செயலாக்குகிறது...",
+	"search_to_get_results": "முடிவுகளைப் பெற தேடவும்",
+	"use_amoled_mode": "கருமை நிற இருண்ட தீம்",
+	"pitch_dark_theme": "AMOLED முறை",
+	"normalize_audio": "ஒலியை சீரமை",
+	"change_cover": "அட்டையை மாற்று",
+	"add_cover": "அட்டையைச் சேர்",
+	"restore_defaults": "இயல்புநிலைகளை மீட்டமை",
+	"download_music_codec": "இசை கோடெக்கை பதிவிறக்கு",
+	"streaming_music_codec": "இசை கோடெக்கை ஸ்ட்ரீம் செய்",
+	"login_with_lastfm": "Last.fm உடன் உள்நுழைக",
+	"connect": "இணை",
+	"disconnect_lastfm": "Last.fm இலிருந்து துண்டி",
+	"disconnect": "துண்டி",
+	"username": "பயனர்பெயர்",
+	"password": "கடவுச்சொல்",
+	"login": "உள்நுழைக",
+	"login_with_your_lastfm": "உங்கள் Last.fm கணக்குடன் உள்நுழைக",
+	"scrobble_to_lastfm": "Last.fm க்கு ஸ்க்ரோபிள் செய்",
+	"go_to_album": "ஆல்பத்திற்குச் செல்",
+	"discord_rich_presence": "Discord செழுமையான தோற்றம்",
+	"browse_all": "அனைத்தையும் உலாவு",
+	"genres": "வகைகள்",
+	"explore_genres": "வகைகளை ஆராயுங்கள்",
+	"friends": "நண்பர்கள்",
+	"no_lyrics_available": "மன்னிக்கவும், இந்தப் பாடலுக்கான பாடல் வரிகளைக் கண்டுபிடிக்க முடியவில்லை",
+	"start_a_radio": "வானொலியைத் தொடங்கு",
+	"how_to_start_radio": "வானொலியை எவ்வாறு தொடங்க விரும்புகிறீர்கள்?",
+	"replace_queue_question": "தற்போதைய வரிசையை மாற்ற விரும்புகிறீர்களா அல்லது அதனுடன் சேர்க்க விரும்புகிறீர்களா?",
+	"endless_playback": "முடிவற்ற இயக்கம்",
+	"delete_playlist": "பாடல் பட்டியலை நீக்கு",
+	"delete_playlist_confirmation": "இந்த பாடல் பட்டியலை நீக்க விரும்புகிறீர்களா?",
+	"local_tracks": "உள்ளூர் பாடல்கள்",
+	"local_tab": "உள்ளூர்",
+	"song_link": "பாடல் இணைப்பு",
+	"skip_this_nonsense": "இந்த அர்த்தமற்றதைத் தவிர்",
+	"freedom_of_music": ""இசையின் சுதந்திரம்"",
```

Comment:
You're using double quotes inside, so it's causing validation issues

Either remove the double quotes or escape them using a `\`